backend/parsing/Automarkable.java

```diff
@@ -0,0 +1,38 @@
+package parsing;
```

Comment:
We'll check the package naming convention for the rest of the KuDoS code. Possibly uk.ac.chu.kudos.parsing... but let's check after the planning meeting this afternoon.